meta/parse.pl

```diff
@@ -1122,6 +1122,9 @@ sub ProcessFunctionSection
 
         $type = $1 if $def =~ /^(\w+) sai_\w+$/;
 
+        LogError "function $name return type should be sai_(status|object_type|object_id)_t, but is $type"
```

Comment:
meta/parse.pl is here because of signoff rebase:
git rebase HEAD~5 --signoff

Reply:
was that removed ?

Reply:
this line exists already on master, its weird that this is present here on diff

Reply:
I forgot "--signoff" option in my last commit caused DCO checking failed.
I followed the DCO instruction to do "git rebase HEAD~5 --signoff" so the meta/parse.pl appeared in my branch. 
Don't know how to remove it for now. Maybe I can close this one and create a separate PR.

Reply:
try force push with squash